File: tests/test_example.js
// Simple JavaScript test file
function greet(name) {
    return `Hello, ${name}!`;
}

export function calculateSum(a, b) {
    return a + b;
}

class Calculator {
    constructor() {
        this.history = [];
    }
    
    add(a, b) {
        const result = a + b;
        this.history.push(`${a} + ${b} = ${result}`);
        return result;
    }
    
    static multiply(x, y) {
        return x * y;
    }
}

export class MathUtils extends Calculator {
    async power(base, exponent) {
        return Math.pow(base, exponent);
    }
}

File: tests/test_example.java
// Simple Java test file
public class Calculator {
    private java.util.List<String> history;
    
    public Calculator() {
        this.history = new java.util.ArrayList<>();
    }
    
    public int add(int a, int b) {
        int result = a + b;
        history.add(a + " + " + b + " = " + result);
        return result;
    }
    
    public static int multiply(int x, int y) {
        return x * y;
    }
    
    protected java.util.List<String> getHistory() {
        return history;
    }
}

interface MathOperations {
    int calculate(int a, int b);
}

enum Operation {
    ADD, SUBTRACT, MULTIPLY, DIVIDE
}